commands/sheriff.js

```diff
@@ -1,10 +1,21 @@
+const Game = require('../src/game.js');
+
 module.exports = {
 	name: 'sheriff',
 	description: 'Create new sheriff game',
 	execute(client, message, args) {
-		console.log('message', message);
-		if(message.mentions.users.size < 5){
-            return message.channel.send('Five players are needed for a game of sheriff');
-        }
+		// console.log('message', message);
+		// if(message.mentions.users.size < 5){
+        //     return message.channel.send('Five players are needed for a game of sheriff');
+        // }
+
+		// Username#1318
+		let creator = `${message.author.username}#${message.author.discriminator}`;
```

Comment:
There's probably a more straightforward way to get this.

Reply:
Can't we use message.author.tag?